src/documents/forms/DocumentTitleValidationIndicators.js

```diff
@@ -38,21 +38,29 @@ const DocumentTitleValidationIndicators = ({errors}) => {
       <List.Item>
         <Icon
           name={
-            (errors.file_name && errors.file_name.file_ext) ||
-            (errors.file_name && errors.file_name.upload_similarity)
+            errors.file_name && errors.file_name.upload_similarity
               ? 'x'
               : 'check'
           }
           color={
-            (errors.file_name && errors.file_name.file_ext) ||
```

Comment:
what are we removing the file extension check here?

Reply:
I moved the extension check to line 54 - line 63